jvm-packages/xgboost4j-spark/src/main/scala/ml/dmlc/xgboost4j/scala/spark/params/GpuParams.scala

```diff
@@ -0,0 +1,53 @@
+/*
+ Copyright (c) 2014 by Contributors
```

Comment:
Is it intended to put this in CPU package?

Reply:
Yes, the para is shared by CPU and GPU, since the entry (XGBoostClassifier/XGBoostRegressor) is located in CPU package